examples/local/otel-config.yaml

```diff
@@ -42,7 +51,7 @@ service:
       processors: [batch]
       exporters: [logging]
     metrics:
-      receivers: [otlp, opencensus, prometheus]
+      receivers: [otlp, opencensus, prometheus, prometheus_discovery]
```

Comment:
Do we need to keep on running `prometheus`? I'd rather leave it out (unless I'm **very** mistaken and we still need it running on the side)

Reply:
The `prometheus` receiver scrapes metrics and the `prometheus_discovery` find targets and generate `present` metrics, so we need to keep them both.

Reply:
Yeah, I see these as independent options. There's probably not much duplication of code, at this point.